frontend/src/types/Crops.ts

```diff
@@ -71,7 +71,7 @@ export type NMPFileSoilTestData = {
   valPH?: number;
   convertedKelownaK?: number;
   convertedKelownaP?: number;
-  sampleDate?: Date;
+  sampleDate?: string;
```

Comment:
Was this change made for the report? I don't see any changes in this PR that would require changing this from a `Date` to a `string`.